src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/repository/TaskResourceCustomRepositoryImpl.java

```diff
@@ -60,6 +61,7 @@ public List<String> searchTasksIds(int firstResult,
             PAGINATION_CLAUSE
         );
 
+        log.debug("Task search query [{}]", queryString);
```

Comment:
might be useful to start this out at `info` level if no sensitive info.

Reply:
Done